helm-values/traction/values-production.yaml

```diff
@@ -1,15 +1,51 @@
 ingressSuffix: -prod.apps.silver.devops.gov.bc.ca
 acapy:
+  ledgers.yml:
+    - id: candy-prod
+      is_production: true
+      is_write: true
+      genesis_url: 'http://test.bcovrin.vonx.io/genesis'
```

Comment:
Question for @esune what would the real Candy Prod ledger genesis url be?

Reply:
See here: https://github.com/bcgov/trust-over-ip-configurations/blob/284153d343e84e23997286672984ac5b74b542a3/openshift/templates/issuer-agent/config/prod/ledgers-candy.yaml#L4

Reply:
This needs to be updated to the actual value - currently fixed manually in `prod` temporarily

Reply:
updated